### PR TITLE
migrate drafts to paging 3

### PR DESCRIPTION
**Pull request by connyduck**

### CODE REVIEW

**Review by charlag — APPROVED** [review]
Wow that looks straightforward

**Comment by connyduck** [comment]
> Wow that looks straightforward

Yeah but I forgot a part 🤦

File: app/src/main/java/com/keylesspalace/tusky/service/SendTootService.kt [hunk 1]
```diff
@@ -244,8 +251,8 @@ class SendTootService : Service(), Injectable {
     }
 
     private fun saveTootToDrafts(toot: TootToSend) {
-
-        draftHelper.saveDraft(
+        serviceScope.launch {
```
Comment (charlag) [inline app/src/main/java/com/keylesspalace/tusky/service/SendTootService.kt:254]: Maybe easier to just block? I think that's how services are supposed to work anyways.

**Review by charlag — APPROVED** [review]

File: app/src/main/java/com/keylesspalace/tusky/service/SendTootService.kt (hunk 1 above)
Comment (connyduck) [reply app/src/main/java/com/keylesspalace/tusky/service/SendTootService.kt:254]: No they run on the main thread 😬

Comment (charlag) [reply app/src/main/java/com/keylesspalace/tusky/service/SendTootService.kt:254]: Ah, true dat, we don't have separate process

**Pull request merged**